src/lib/components/auth/Auth0Button.svelte

```
@@ -1,12 +1,28 @@
 <script lang="ts">
+  import { onMount } from 'svelte';
   export let className = '';
+  
+  // Create an absolute URL using the current origin
+  function createFullUrl(path: string): string {
```

ESLint

[error] 'createFullUrl' is defined but never used.

(@typescript-eslint/no-unused-vars)